app/main.ipynb

```diff
@@ -216,15 +282,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "A.mean(axis=1)\n",
```

Comment:
This line is supposed to calculate the standard deviation by the y-axis, but it currently calculates the mean. You should use `A.std(axis=1)` instead of `A.mean(axis=1)` to get the standard deviation.